rpc/params/storage.go

```diff
@@ -913,6 +913,10 @@ type ImportStorageParams struct {
 
 	// StorageName is the name of the storage to assign to the entity.
 	StorageName string `json:"storage-name"`
+
+	// Force indicates whether to force the import operation when there are
+	// conflicting storage resources that would otherwise prevent the import.
+	Force bool `json:"force,omitempty"`
```

Comment:
We would introduce a `ImportStorageParamsV2` struct with the force attribute.
v6 api takes the original struct, v7 api takes the new struct.
There'd be some internal refactoring to get the code reuse happening

Reply:
Applied the refactor in the latest change.